### DISCUSSION TITLE
rename `master` branch to `main`

### DISCUSSION THREAD

**Discussion by longlho** [ember-intl/ember-intl#1524] [Ideas]
`master` as a history of being used in the context of slavery. GitHub also sets the default branch as `main` when you create a new repo already. We should rename the main branch.

Thoughts?

**Comment by buschtoens** [comment]
Much in favor! I long wanted to do this, but kept putting it off as to not accidentally break things.

**Reply by longlho** [reply]
hmm I pushed `main` but cannot switch default branch. @jasonmit can u give me perm?

**Reply by jasonmit** [reply]
@longlho made you an admin, feel free to make the update!
